tests/should-pass/iati-activities/iati-activity/result/reference/no-reference-element-present.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
Is this needed? Isn't it covered by the minimal IATI file case?

Reply:
Nvm, leave for integration branch as needs to be moved to `result` level folder.

Reply:
Fixed in e23a674 anyway!